web-console/future-openshift/cron-jobs/cron-jobs.md

```diff
@@ -0,0 +1,27 @@
+# Cron Jobs
+
+## Create Cron Job
+![](img/cron-job-list.png)
```

Comment:
super nit, but the "Create Cron Job" button looks like there is not space between Cron & Job

Reply:
Good catch; will update

Reply:
No space should be between Cron and Job, the resource name is CronJob written together, there's a separate resource Job, so adding that space between might be confusing to users.